lib/dry/system/container.rb

```diff
@@ -634,14 +636,35 @@ def importer
           @importer ||= config.importer.new(self)
         end
 
-        # Registers a callback hook to run afer container lifecycle events.
+        # Registers a callback hook to run before container lifecycle events.
```

Comment:
Might just leave these as is, since in the docs they'll be clearly connected to the method name, and to me the bolding only really makes sense if the terms "before" and "after" are closer to each other than they actually are in this file.